### DISCUSSION TITLE
Update translation more regularly

### DISCUSSION THREAD

**Discussion by hahahahacker2009** [element-hq/element-meta#1765] [Ideas]
Translations aren't updated for weeks. There always "alerts" on weblate that there are so many changes compared to upstream. There is some language which have many contributors, they can make a suggestion instead saving the translation directly like languages with only one man contribute to it now.

**Comment by t3chguy** [comment]
Saving the translation directly means it doesn't get vetted by the release maintainer, and thus allow people to inject bad or malicious translations. The Element Web & Desktop project pull from Weblate on every release, what's the benefit of doing it more frequently than every release?

**Reply by hahahahacker2009** [reply]
Update seems too slow.
If we deliver translation more quickly, this would allow translators to test if their translations is good, or needs improvements. 
If there is bad/malicious translation, I don't think the translation administrators will really found them (if they don't know the language). Languages with more contributors would revert the changes more quickly. 
Also,, do we need to rebuild the app to update translations?

**Reply by t3chguy** [reply]
> Also,, do we need to rebuild the app to update translations?

Yes

**Reply by hahahahacker2009** [reply]
Oh, I completely forgot this point.

**Comment by hahahahacker2009** [comment]
There is another version of Element released, but translations aren't updated.

**Reply by t3chguy** [reply]
Yes this latest release was a hotfix, it carried only a single commit, it went through less testing so we couldn't include translation changes as they may remove strings which haven't been removed in the hotfix release.
